### PR TITLE
Add add-IOI-1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, there is no adventure.

**What is the new behavior (if this is a feature change)?**
- An unforgettable adventure has been added.

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks clean to me.

**Review by LucaAllmaier — APPROVED** [review]
Looks good to me.

File: src/components/pages/Robinett/index.jsx [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+                <MDBCol md="4">
+                  <MDBCard className="border">
+                    <MDBCardBody className="text-left">
+                      <div className="d-flex justify-content-between align-items-center">
+                        <p className="font-weight-bold lead mb-0">Developer</p>
+                        <MDBIcon icon="code" className="blue-text" />
+                      </div>
+                      <p className="text-muted mb-0">
+                        <small>
+                          You must be a developer! Who else would do that?!
```
Comment (pinterid) [inline src/components/pages/Robinett/index.jsx]: Should be translated to German, to keep the language of the site consistent.

File: src/components/pages/Robinett/index.jsx [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+                </MDBCol>
+                <MDBCol md="4">
+                  <MDBCard className="border">
+                    <MDBCardBody className="text-left">
+                      <div className="d-flex justify-content-between align-items-center">
+                        <p className="font-weight-bold lead mb-0">Explorer</p>
+                        <MDBIcon icon="map-signs" className="orange-text" />
+                      </div>
+                      <p className="text-muted mb-0">
+                        <small>You found this page. Wow.</small>
```
Comment (pinterid) [inline src/components/pages/Robinett/index.jsx]: Should be translated to German, to keep the language of the site consistent.

File: src/components/pages/Robinett/index.jsx [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+import MDBSortable from "mdb-react-sortable";
+
+//> Components
+import { Egg } from "../../atoms";
+
+//> CSS
+import "./robinett.scss";
+
+//> Data
+const Item = (props) => <span className={props.className}>{props.title}</span>;
```
Comment (schettn) [inline src/components/pages/Robinett/index.jsx:32]: According to ESLINT arrow functions are no capitalised.
```suggestion
const item = (props) => <span className={props.className}>{props.title}</span>;
```
Btw do not forget to change the function call parts in the code.

File: src/components/pages/Robinett/index.jsx [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+                        <p className="font-weight-bold lead mb-0">
+                          Cave explorer
+                        </p>
+                        <MDBIcon
+                          icon="angle-double-down"
+                          className="blue-text"
+                        />
+                      </div>
+                      <p className="text-muted mb-0">
+                        <small>You really want to go down there, right?</small>
```
Comment (pinterid) [inline src/components/pages/Robinett/index.jsx]: Should be translated to German, to keep the language of the site consistent.

**Review by schettn — CHANGES_REQUESTED** [review]
Check before merging

**Review by pinterid — CHANGES_REQUESTED** [review]
Looks good after a few changes.

File: src/components/pages/Robinett/index.jsx (hunk 3 above)
Comment (schettn) [reply src/components/pages/Robinett/index.jsx:32]: According to ESLINT, EsLiNt is spelled ESLint. Uff

Comment (Aichnerc) [reply src/components/pages/Robinett/index.jsx:32]: This is an component element, so it's meant to be uppercase. (See how it is called in line 36)

**Review by pinterid — APPROVED** [review]
Looks good to merge.

File: src/components/pages/Robinett/index.jsx (hunk 3 above)
Comment (schettn) [reply src/components/pages/Robinett/index.jsx:32]: I see. Nvm.

**Review by schettn — APPROVED** [review]
🚀

**Pull request merged**